packages/dd-trace/src/config.js

```diff
@@ -141,6 +141,17 @@ class Config {
       false
     )
 
+    const DD_TRACE_STATS_COMPUTATION_ENABLED = coalesce(
```

Comment:
According to the RFC, we're supposed to have an opt-in beta period, so it should be behind this flag.

Reply:
Also, we seem to currently only support protocol versions 0.4 and 0.5. Setting the value to 0.6 breaks the trace exporter because it thinks anything not 0.5 should be 0.4. 😐 

Is the reason we don't have a separate exporter for 0.6 because everything in it is backwards compatible with 0.5? If so, I could change the logic for that to not think a protocol version of 0.6 should be using the 0.4 encoder. 😅

Reply:
I think 0.6 has a different format with trace chunk metadata being outside of spans. We could always use a mapping to force 0.5 for traces when 0.6 is used until it's implemented, but we should still respect the config.

Reply:
We could, but the RFC still says this should be behind a flag and have a beta period before switching it on by default. There's also use cases for turning it off given that this will drop data on individual traces when we set it up to drop p0 traces, so either way I'd say it should still be using the flag like this.